camunda.Dockerfile

```diff
@@ -4,82 +4,17 @@
 # see https://docs.docker.com/build/buildkit/#getting-started
 # Both ubuntu and eclipse-temurin are pinned via digest and not by a strict version tag, as Renovate
 # has trouble with custom versioning schemes
-ARG BASE_IMAGE="ubuntu:noble"
-ARG BASE_DIGEST="sha256:a08e551cb33850e4740772b38217fc1796a66da2506d312abe51acda354ff061"
-ARG JDK_IMAGE="eclipse-temurin:21-jdk-noble"
-ARG JDK_DIGEST="sha256:c04e695e59a97337e87d55ebbe9f527aacec1504b78ffac2730475057a8ea465"
+ARG BASE_IMAGE="reg.mini.dev/openjre:21-dev"
+ARG BASE_DIGEST="sha256:80e113714c426176d4c8159d0729bfbe799784f04e4efac4b376a9f17e0fa689"
 
 # set to "build" to build camunda from scratch instead of using a distball
 ARG DIST="distball"
 
-### Base image ###
-# All package installation, updates, etc., anything with APT should be done here in a single step
-# hadolint ignore=DL3006
-FROM ${BASE_IMAGE}@${BASE_DIGEST} AS base
-WORKDIR /
-
-# Use custom APT timeout and retry values for more resilient builds
-COPY .github/actions/build-platform-docker/99apt-timeout-and-retries /etc/apt/apt.conf.d/
-
-# Upgrade all outdated packages and install missing ones (e.g. locales, tini)
-# This breaks reproducibility of builds, but is acceptable to gain access to security patches faster
-# than the base image releases
-# FYI, installing packages via APT also updates the dpkg files, which are few MBs, but removing or
-# caching them could break stuff (like not knowing the package is present) or container scanners
-# hadolint ignore=DL3008
-RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
-    --mount=type=cache,target=/var/lib/apt,sharing=locked \
-    --mount=type=cache,target=/var/log/apt,sharing=locked \
-    apt-get -qq update && \
-    apt-get install -yqq --no-install-recommends tini ca-certificates systemd perl && \
-    apt-get upgrade -yqq --no-install-recommends systemd perl
-
-### Build custom JRE using the base JDK image
-# hadolint ignore=DL3006
-FROM ${JDK_IMAGE}@${JDK_DIGEST} AS jre-build
-
-# Build a custom JRE which will strip down and compress modules to end up with a smaller Java \
-# distribution than the official JRE. This will also include useful debugging tools like
-# jcmd, jmod, jps, etc., which take little to no space. Anecdotally, compressing modules add around
-# 10ms to the start up time, which is negligible considering our application takes ~10s to start up.
-# See https://adoptium.net/blog/2021/10/jlink-to-produce-own-runtime/
-# hadolint ignore=DL3018
-# required to compile a JRE on ARM64
-# see https://github.com/openzipkin/docker-java/issues/34
-ENV JAVA_TOOL_OPTIONS "-Djdk.lang.Process.launchMechanism=vfork"
-RUN jlink \
-     --add-modules ALL-MODULE-PATH \
-     --strip-debug \
-     --no-man-pages \
-     --no-header-files \
-     --compress=2 \
-     --output /jre && \
```

Comment:
❓ Why aren't we building our own JRE anymore?
❓ Does the new JRE include tools like jps, jcmd, etc.?

Reply:
They don't, no. That would be a good point to build our own JRE from the JDK image, allowing us to keep these utils. I think long term I would prefer not doing that and having a debug image instead (to use with `kubectl debug` and `docker debug`), but we could keep it right now for this iteration.

Reply:
> I think long term I would prefer not doing that and having a debug image instead

Agree

Reply:
I gave it a quick try, unfortunately you run into some issues when building the JRE due to missing libraries, namely `libz`, which prevents you from using CDS :|

While we don't need the `-Xshare:dump` anymore (thanks to JEP-341), you cannot use `-Xshare:auto` either :(

I don't think it's a deal breaker now, but it would also prevent us from looking into AppCDS in the future to speed up the startup...